getting_started/setup_vm/roles/openssl/vars/common.yml

```
@@ -0,0 +1,4 @@
+# As per https://launchpad.net/ubuntu/+source/openssl/1.1.1f-1ubuntu2.18,
+# this corresponds to OpenSSL 1.1.1t, including patches for CVE-2023-0286,
+# CVE-2023-0215, CVE-2022-4450 and CVE-2022-4304.
+openssl_ubuntu_deb_package: "openssl=1.1.1f-1ubuntu2.18"
```

```suggestion
openssl_ubuntu_deb_package: "openssl>=1.1.1f-1ubuntu2.18"
```

This is otherwise going to break very quickly at best, or result in outdated versions of openssl installed in images at worst.

---

I'm not sure. First, the `>=` syntax doesn't seem to work out of the box with Ansible's `apt`. Second, `openssl` is a critical dependency that is pinned on SGX (via Open Enclave), and so should also be on SNP builds? I think there's only one version that's ever allowed here (the very latest) at any given point in time? Happy to be convinced otherwise though!

---

Two things:

1. It's pinned on SGX by necessity, not by design. We do not want to prevent our users from upgrading.
2. -1ubuntu2.18 is an extremely ephemeral build suffix in the ubuntu apt repo. There may be a 2.19, 2.20, and after a few weeks 2.18 will be gone from the repo (only a few binaries are kept), and the playbook will start failing. We definitely do not want to update the playbook and our images that often.

I think we really want to express "1.1.1f and above", as much as possible.

---

As suggested, we now check this via `cpack` rather than Ansible.